bot.js

```diff
@@ -69,29 +69,35 @@ function matchPattern(p, w, pi, wi) {
 function saySomething() {
   var message = document.getElementById('message').value;
   var conversation = document.getElementById('conversation');
-  var answer = "Can you tell me more about that?";
-  var words = message.split(' ')
+  reply(message);
```

Comment:
This works fine, but I would prefer a version where `reply` returns the reply and `saySomething` does the UI handling, i.e.:

```
var message = document.getElementById('message').value;
var conversation = document.getElementById('conversation');
var answer = reply(message);
conversation.innerHTML += "<p><strong>You:</strong> " + message + "</p>";
	  conversation.innerHTML += "<p><strong>Eliza:</strong> " + answer + "</p>";